### ISSUE TITLE
Keybase fs stopped working

### ISSUE THREAD

**Issue by ismay** [keybase/client#13541]
<img width="811" alt="screen shot 2018-08-31 at 18 02 34" src="https://user-images.githubusercontent.com/7355199/44923288-1799b500-ad48-11e8-804b-ea8b4df93c04.png">

Keybase fs stopped working all of a sudden on my mac running macOS 10.13.6 and a fresh install of keybase (through brew):

<img width="303" alt="screen shot 2018-08-31 at 18 03 52" src="https://user-images.githubusercontent.com/7355199/44923342-3ef08200-ad48-11e8-9f1a-8219a0b99b00.png">

It worked a couple of minutes ago.

**Comment by ismay** [comment]
@strib Thanks for the reply. Just uninstalled the brew version and installed the version on the website, but I still get the same error.

**Comment by strib** [comment]
Sounds like you need to fully quit and restart all parts of the Keybase app.  Maybe a system reboot would help get rid of any old brew-based processes that are still running.

**Comment by ismay** [comment]
Allright. I'll try that. I'll let you know how it went.

**Comment by ismay** [comment]
Same thing. After rebooting I get the exact same error, even with the new client from the keybase website (after removing the brew version before the reboot): 

<img width="822" alt="screen shot 2018-08-31 at 18 33 25" src="https://user-images.githubusercontent.com/7355199/44924721-8547e000-ad4c-11e8-9c16-6839c9eec3a1.png">

**Comment by ismay** [comment]
Is it persisting settings somewhere that I need to delete as well?

**Comment by ismay** [comment]
Ok it was. Used `brew cask zap keybase` which removes all associated files, reinstalled the app from the website and now it mounts again.

What's weird though is that the reported version is exactly the same as the version that was installed through brew. This is the version I get from the app now (downloaded from keybase.io):

<img width="301" alt="screen shot 2018-08-31 at 18 45 11" src="https://user-images.githubusercontent.com/7355199/44925197-0358b680-ad4e-11e8-93e1-1a05c48f6619.png">

Same as the brew version right? Are you sure it's outdated?

**Comment by strib** [comment]
The KBFS process is separate from the GUI process.  I don't know who's maintaining brew these days, but maybe their kbfs binary is out of date.  It's the kbfs process that was giving the error you were seeing.

**Comment by ismay** [comment]
Ah I didn't know. So in that case I assume kaybase also cannot deprecate the brew version, because you're not maintaining it? Because I had no idea it was shipping with out of date parts.

**Comment by strib** [comment]
Yeah sorry, I have no idea who maintains it.  The website lists it as "Inactive" if that means anything: http://brewformulas.org/keybase

**Comment by ismay** [comment]
I guess this is where it comes from: [https://github.com/Homebrew/homebrew-cask/blob/master/Casks/keybase.rb](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/keybase.rb)? Maybe it would be possible to deprecate or update it via a PR there.

**Comment by ismay** [comment]
Just got the exact same error again. Happened while working on some files, where all of a sudden kbfs died, and now it's giving me these errors again. So it's not related to me using an older version, but apparently just something that causes it to crash and doesn't allow it to recover. Even after reboots. The only thing that is a known solution for me is removing keybase, the device and reinstalling and reregistering the device with keybase.

So definitely a bug. A pretty catastrophic one at that.

**Comment by strib** [comment]
Seems like something on your system is re-installing the old version?  Or it's still running somehow?  When that happens, you can look around for running `kbfs` processes and/or do a `keybase log send` and we can see if there's anything obvious wrong.
